frontend/src/component/playground/Playground/PlaygroundEnvironmentTable/PlaygroundEnvironmentDiffTable.test.tsx

```diff
@@ -0,0 +1,65 @@
+import { screen } from '@testing-library/react';
+import { render } from 'utils/testRenderer';
+import { PlaygroundEnvironmentDiffTable } from './PlaygroundEnvironmentDiffTable';
+import { UIProviderContainer } from '../../../providers/UIProvider/UIProviderContainer';
+
+const irrelevantDetails = {
+    strategies: {
+        data: [],
+        result: false,
+    },
+    isEnabledInCurrentEnvironment: true,
+    variants: [],
+    variant: {
+        name: 'variantName',
+        enabled: true,
+        payload: {
+            type: 'string' as const,
+            value: 'variantValue',
+        },
+    },
+    projectId: 'projectA',
+};
+
+test('should render environment table', async () => {
+    render(
+        <UIProviderContainer>
+            <PlaygroundEnvironmentDiffTable
+                features={{
+                    development: [
+                        {
+                            name: 'featureA',
+                            isEnabled: true,
+                            environment: 'development',
+                            context: {
+                                channel: 'web',
+                                client: 'clientA',
+                                appName: 'myapp',
+                            },
+                            ...irrelevantDetails,
```

Comment:
I use this to separate data that matters to this test from data that only satisfies a type signature